content/docs/ops/maintenance-list.md

```diff
@@ -5,48 +5,73 @@ menu:
 title: Ongoing platform maintenance
 ---
 
-The platform requires regular support and maintenance activities to remain in a compliant state. If you are on support and you can’t complete any of the items personally, you are responsible for ensuring that an appropriate person does it.
+The platform requires regular support and maintenance activities to remain in a 
+compliant state. If you are on support and you can’t complete any of the items 
+personally, you are responsible for ensuring that an appropriate person does it.
 
 # Is it your first day of support?
 
-- Update the [`#cg-platform`](https://gsa-tts.slack.com/messages/cg-platform/) topic to include your name as the support contact.
-- Update the support schedule by moving yourself to the end of the list in the [slackbot auto response](https://gsa-tts.slack.com/customize/slackbot) for `atlas support schedule`.
-- Join/unmute [`#cg-support`](https://gsa-tts.slack.com/messages/cg-support/) and [`#cg-platform-news`](https://gsa-tts.slack.com/messages/cg-platform-news/)
-- Meet with the previous support person and take responsibility for any open support items they are still working on.
-
+- Update the [`#cg-platform`](https://gsa-tts.slack.com/messages/cg-platform/) 
+topic to include your name as the support contact.
+- Update the support schedule by moving yourself to the end of the list in the 
+[slackbot auto response](https://gsa-tts.slack.com/customize/slackbot) for 
+`atlas support schedule`.
+- Join/unmute [`#cg-support`](https://gsa-tts.slack.com/messages/cg-support/) 
+and [`#cg-platform-news`](https://gsa-tts.slack.com/messages/cg-platform-news/)
+- Meet with the previous support person and take responsibility for any open 
+support items they are still working on.
 
 # Daily maintenance checklist
 
 The tasks on this checklist should be performed each day.
 
+## PR as you go
+
+If you see a way to make this checklist better, just submit a PR to the 
+[cg-site](https://github.com/18F/cg-site) repo.
+
 ## Ensure all VMs are running the current stemcell
 
 - Get the latest stemcell version from http://bosh.cloudfoundry.org/stemcells/.
 
-- Check https://ci.fr.cloud.gov/teams/main/pipelines/aws-light-stemcell-builder to ensure it has built the same version.
+- Check https://ci.fr.cloud.gov/teams/main/pipelines/aws-light-stemcell-builder 
+to ensure it has built the same version.
 
-- On each bosh director run `bosh deployments` and verify the stemcell in-use for each deployment is current.
+- On each bosh director run `bosh deployments` and verify the stemcell in-use 
+for each deployment is current.
```

Comment:
We're talking about having a pipeline task for this. But it shouldn't stop this update. :+1:

for your eyes, https://gsa-tts.slack.com/archives/C0ENP71UG/p1496895499351167